diff --git a/docroot/modules/dblog/dblog.install b/docroot/modules/dblog/dblog.install
index c2e4119..2531bef 100644
--- a/docroot/modules/dblog/dblog.install
+++ b/docroot/modules/dblog/dblog.install
@@ -163,6 +163,15 @@ function dblog_update_7003() {
   }
 }
 
+function dblog_modules_enabled($modules) {
+  if (in_array('dblog', $modules) && !empty($_ENV['AH_SITE_ENVIRONMENT']) &&  $_ENV['AH_SITE_ENVIRONMENT'] == '01live') {
+    drupal_set_message(t('Syngenta does not allow the use of "Database logging" on production, please stream line
+       your logs from cloud.acquia.com.'), 'error');
+    module_disable(array('dblog'));
+    drupal_uninstall_modules(array('dblog'));
+  }
+}
+
 /**
  * @} End of "addtogroup updates-7.x-extra".
  */
