diff --git a/modules/dblog/dblog.install b/modules/dblog/dblog.install
index c2e4119..037a2f4 100644
--- a/modules/dblog/dblog.install
+++ b/modules/dblog/dblog.install
@@ -138,6 +138,16 @@ function dblog_update_7001() {
   ));
 }
 
+function dblog_modules_enabled($modules) {
+  if (in_array('dblog', $modules) && !empty($_ENV['AH_SITE_ENVIRONMENT']) &&  $_ENV['AH_SITE_ENVIRONMENT'] == '01live') {
+    drupal_set_message(t('Syngenta does not allow the use of "Database logging" on production, please stream line
+       your logs from cloud.acquia.com.'), 'error');
+    module_disable(array('dblog'));
+    drupal_uninstall_modules(array('dblog'));
+  }
+}
+
+
 /**
  * @} End of "addtogroup updates-6.x-to-7.x".
  */
@@ -163,6 +173,7 @@ function dblog_update_7003() {
   }
 }
 
+
 /**
  * @} End of "addtogroup updates-7.x-extra".
  */
